src/ts/components/playbacktimelabel.ts

```diff
@@ -133,7 +133,7 @@ export class PlaybackTimeLabel extends Label<PlaybackTimeLabelConfig> {
     };
 
     player.on(player.exports.PlayerEvent.TimeChanged, playbackTimeHandler);
-    player.on(player.exports.PlayerEvent.Seeked, playbackTimeHandler);
+    player.on(player.exports.PlayerEvent.Seek, playbackTimeHandler);
```

Comment:
Would it hurt to have both? Due to possible `time` adjustment in the player. Not sure how often `timechanged` event would not come right after `seeked` 🤷‍♂️

Reply:
Makes sense. 👍